e2e-tests/insurance/cypress/e2e/journeys/eligibility/member-of-a-group/member-of-a-group.spec.js

```diff
@@ -0,0 +1,145 @@
+      cy.completeEndBuyerForm();
+      cy.completePartyToConsortiumForm();
+
+      cy.assertUrl(url);
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+    });
+
+    it('renders core page elements', () => {
+      cy.corePageChecks({
+        pageTitle: CONTENT_STRINGS.PAGE_TITLE,
+        currentHref: MEMBER_OF_A_GROUP,
+        backLink: PARTY_TO_CONSORTIUM,
+        assertAuthenticatedHeader: false,
+      });
+    });
+
+    describe('page tests', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it('renders `yes` radio button', () => {
+        yesRadio().input().should('exist');
+
+        cy.checkText(yesRadio().label(), FIELD_VALUES.YES);
+      });
+
+      it('renders `no` radio button', () => {
+        cy.checkText(noRadio().label(), FIELD_VALUES.NO);
+
+        cy.checkRadioInputNoAriaLabel(CONTENT_STRINGS.PAGE_TITLE);
+      });
+
+      describe(`expandable details -${MEMBER_OF_A_GROUP_DESCRIPTION.INTRO}`, () => {
+        it('renders summary text', () => {
+          cy.checkText(memberOfAGroupPage.summary(), MEMBER_OF_A_GROUP_DESCRIPTION.INTRO);
+
+          memberOfAGroupPage.details().should('not.have.attr', 'open');
+        });
+
+        describe('when clicking the summary text', () => {
+          it('should expand the collapsed `details` content', () => {
+            memberOfAGroupPage.summary().click();
+            memberOfAGroupPage.details().should('have.attr', 'open');
+
+            cy.checkText(memberOfAGroupPage.description(), MEMBER_OF_A_GROUP_DESCRIPTION.DESCRIPTION);
+          });
+        });
+      });
+    });
+
+    describe('when submitting an empty form', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it('should render validation errors', () => {
+        const expectedErrorsCount = 1;
+
+        cy.submitAndAssertRadioErrors({
+          field: yesRadio(FIELD_ID),
+          expectedErrorsCount,
+          expectedErrorMessage: ERROR_MESSAGES.INSURANCE.ELIGIBILITY[FIELD_ID].IS_EMPTY,
+        });
+      });
+    });
+
+    describe('when submitting the answer as `No`', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+
+        cy.clickNoRadioInput();
+        cy.clickSubmitButton();
+      });
+
+      it(`should redirect to ${CHECK_YOUR_ANSWERS}`, () => {
+        const expectedUrl = `${baseUrl}${CHECK_YOUR_ANSWERS}`;
+
+        cy.assertUrl(expectedUrl);
+      });
+
+      describe('when going back to the page', () => {
+        it('should have the originally submitted answer selected', () => {
+          cy.clickBackLink();
+
+          cy.assertNoRadioOptionIsChecked();
+        });
+      });
+    });
+
+    describe('when submitting the answer as `yes`', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+
+        cy.clickYesRadioInput();
+        cy.clickSubmitButton();
+      });
+
+      it(`should redirect to ${LONG_TERM_COVER}`, () => {
```

Comment:
Looks like this should go to "check your answers", when looking at the order in Figma?